libs/common/src/models/domain/organization.ts

```diff
@@ -196,8 +199,22 @@ export class Organization {
     return this.canManagePolicies;
   }
 
-  get canManageBilling() {
-    return this.isOwner && (this.isProviderUser || !this.hasProvider);
+  get canViewSubscription() {
+    return this.hasProvider && this.providerType === ProviderType.Msp
+      ? this.isProviderUser
+      : this.isOwner;
+  }
```

Comment:
This is what I meant in my previous comment: Edit implies (gives permissions to) View

```suggestion
  get canViewSubscription() {
    if (this.canEditSubscription) {
      return true;
    }
    
    return this.hasProvider && this.providerType === ProviderType.Msp
      ? this.isProviderUser
      : this.isOwner;
  }
```

I think this is cleaner than the other way around, where Edit might fail because they don't have View. Instead, assume that if they can Edit then they can View.

Reply:
That makes perfect sense!